fastlane/lanes/release.rb

```diff
@@ -88,14 +132,54 @@
   #####################################################################################
   desc 'Updates a release branch for a new beta release'
   lane :new_beta_release do |options|
-    android_betabuild_prechecks(base_version: options[:base_version], skip_confirm: options[:skip_confirm])
+    # Checkout default branch and update
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    # Check local repo status
+    ensure_git_status_clean
+
+    # Check versions
+    message = <<-MESSAGE
+
+      Current beta version: #{current_beta_version}
+      New beta version: #{next_beta_version}
+
+      Current build code: #{current_build_code}
+      New build code: #{next_build_code}
+
+    MESSAGE
+
+    # Check branch
+    unless !options[:base_version].nil? || Fastlane::Helper::GitHelper.checkout_and_pull(release: current_release_version)
```

Comment:
I don't think we are using `base_version` option anywhere to trigger this lane. Should we use this opportunity to drop it?

Reply:
@oguzkocer Do you ever use it for hotfixes? I seem to recall needing to use it on other apps once in a while. Maybe I'm misremembering. But yep, I am definitely open to removing it if you don't think we need it anymore.

Reply:
Yeap, we use a base version for the hotfix, but this is the `new_beta_release` lane, so I don't think it'll matter?

Reply:
@oguzkocer I was thinking that there could be a circumstance where a new hotfix build is needed, but you're right--because the version is bumped before hotfix PRs are merged into the branch and a build isn't kicked off until after that, this isn't needed.